absl/utility/CMakeLists.txt

```diff
@@ -25,13 +25,9 @@ list(APPEND UTILITY_SRC
   ${UTILITY_PUBLIC_HEADERS}
```

Comment:
does this UTILITY_SRC need to be defined?  We don't have it in algorithm and it seems redundant.

Reply:
@JonathanDCohen Indeed it is not needed. Unlike Bazel which requires header dependency to be stated in `BUILD` file explicitly, CMake will just let the target build system to figure this out themselves.

Reply:
Could you remove this to as part of the PR?

Reply:
Of course. It was an oversight on my part to leave it in.